webserver/grpc/src/main/java/io/helidon/webserver/grpc/GrpcConfigBlueprint.java

```diff
@@ -53,12 +57,22 @@ interface GrpcConfigBlueprint extends ProtocolConfig {
 
     /**
      * Whether to support compression if requested by a client. If explicitly
-     * disabled, no compression will be ever be used by the server even if a
+     * disabled, no compression will ever be used by the server even if a
      * client-compatible compressor is found.
      *
      * @return true if compression is enabled
      */
     @Option.Configured
     @Option.DefaultBoolean(true)
     boolean enableCompression();
+
+    /**
+     * gRPC server services. These services will not be discovered automatically.
+     *
+     * @return services to use
+     */
+    @Option.Singular
+    @Option.Configured
+    @Option.Provider(value = GrpcServerServiceProvider.class, discoverServices = false)
```

Comment:
Why is `discoverServices` set to `false`? We only want explicitly configured services to be added?